templates/tools/dockerfile/git_config.include

```diff
@@ -0,0 +1,19 @@
+RUN git config --global --add safe.directory /var/local/jenkins/grpc
```

Comment:
I think we do need find a way to simplify this file, this can be done in a follow up PR. Can you add a TODO here?

Reply:
Done, have also added a note on how it can be solved